$ bash -n progress/R44b/tools/run_with_monitor.sh progress/R44b/tools/build_ios_spike.sh
[exit_code=0]
$ diff -u codes/llvm/libcxx/src/ios.cpp tmp/R44b/experiment/libcxx/src/ios.cpp
--- codes/llvm/libcxx/src/ios.cpp	2026-08-05 11:41:57.705122452 +0800
+++ tmp/R44b/experiment/libcxx/src/ios.cpp	2026-08-15 17:30:10.759416416 +0800
@@ -9,6 +9,7 @@
 #include <__config>
 #include <__locale>
 #include <algorithm>
+#include <cxxabi.h>
 #include <ios>
 #include <limits>
 #include <memory>
@@ -362,6 +363,12 @@
 void ios_base::__set_badbit_and_consider_rethrow() {
   __rdstate_ |= badbit;
 #if _LIBCPP_HAS_EXCEPTIONS
+  try {
+    throw;
+  } catch (const __cxxabiv1::__forced_unwind&) {
+    throw;
+  } catch (...) {
+  }
   if (__exceptions_ & badbit)
     throw;
 #endif // _LIBCPP_HAS_EXCEPTIONS
[exit_code=1]
$ sha256sum codes/llvm/libcxx/src/ios.cpp tmp/R44b/experiment/libcxx/src/ios.cpp
1c26a8fe9e565af612953459c6f0f3e4a2a9abde47f76ab5eecff470812c1dfe  codes/llvm/libcxx/src/ios.cpp
7f4600cd03a56cc8c872bdf866b51ae941869f3b2489bad5a6eb6885b88e4123  tmp/R44b/experiment/libcxx/src/ios.cpp
[exit_code=0]
